filters/f_hwtransfer.c

```diff
@@ -165,12 +165,18 @@ static void process(struct mp_filter *f)
                 MP_ERR(f, "no hw upload format found\n");
                 goto error;
             }
+            // Disabled because in the case of the dmabuf-wayland VO, the source image
+            // format may not be one of the formats advertised by the vo as supported,
+            // and yet if it is uploaded to a VAAPI surface, then the vo can display
+            // the frame by extracting the underlying drmprime buffer.
+            /*
```

Comment:
Ideally, you get the vo to advertise formats properly which means getting the ra based format probing in hwdec_vaapi to work. We can't just comment this out.

Reply:
@farfel: Just a small ping on this one in case it got lost (everything else was addressed I think). I can't remember if fixing this depended on one of @philipl's PRs or not.

Reply:
It doesn't depend on my PRs, but to get the most benefit from proper format detection, you need my PRs.